bittensor/_blacklist/__init__.py

```diff
@@ -113,7 +113,10 @@ def blacklist(
             and is_registered
         ):
             uid = metagraph.hotkeys.index(src_hotkey)
-            return metagraph.neurons[uid].validator_permit
+            # Return False (pass) if there is a permit, and True (fail) if there isn't.
+            if metagraph.neurons[uid].validator_permit:
+                return False, "has vpermit"
+            return True, "no vpermit"
```

Comment:
is this a fix? seems like its the negation of what we had, is it?

Reply:
Yep, this is a fix. Previously we'd return True and erroneously blacklist a valid request from a `src_hotkey` with a vpermit.

Likely this doesn't affect many miners because often they roll their own blacklist functions.